smack-extensions/src/main/java/org/jivesoftware/smackx/usertune/provider/UserTuneProvider.java

```diff
@@ -67,10 +69,14 @@ public UserTuneElement parse(XmlPullParser parser, int initialDepth, XmlEnvironm
                     	parser.next();
                     	builder.setTrack(Integer.parseInt(parser.nextText()));
```

Comment:
I'm not sure, if this will work in real life. You have to check, what name the current element has. Right now you are randomly assigning `parser.nextText()` to your elements. If however, the XML is not in the order you expect, this will fail.